packages/app/src/app/pages/Dashboard/index.js

```diff
@@ -52,7 +52,11 @@ class Dashboard extends React.Component {
     } = this.props;
     const { showSidebar } = this.state;
 
-    history.listen(() => {
+    history.listen(({ state }) => {
+      if (!!state && state.from === 'sandboxSearchFocused') {
```

Comment:
Try using the `Boolean` function instead of double negation

```suggestion
      if (Boolean(state) && state.from === 'sandboxSearchFocused') {
```